editor/project_manager.cpp

```diff
@@ -1753,6 +1753,10 @@ void ProjectManager::_notification(int p_what) {
 
 			Engine::get_singleton()->set_editor_hint(false);
 		} break;
+		case NOTIFICATION_RESIZED: {
+
+			open_templates->popup_centered_minsize();
```

Comment:
That seems weird, it will popup whenever you resize the window, even if it was not shown previously, no?

Reply:
oh that's true xD I was testing in a system without projects so I didn't notice it was still showing after having any project. I'll fix it.